src/components/namelist/nameList.js

```diff
@@ -0,0 +1,76 @@
+import { useState, useEffect } from "react";
+import { get } from "../../service/apiClient";
+import "./nameList.css";
+import ProfileCircle from "../profileCircle";
+
+function NameList(inputText) {
+  const [users, setUsers] = useState([]);
+
+  useEffect(() => {
+    const getUserInfo = async () => {
+      const res = await get(`users`);
+      setUsers(res.data.users);
+    };
+    getUserInfo();
+  }, []);
+
+  //    {
+  //     users.map((user) => {
+  //       const userInitials =
+  //         user.firstName.match(/\b(\w)/g) + user.lastName.match(/\b(\w)/g);
+
+  //       return (
+  //         <section className="create-post-user-details student-view">
+  //           <ProfileCircle initials={userInitials} />
+  //           <p>
+  //             {user.firstName}
+  //             {user.lastName}
+  //           </p>
+  //           <div className="edit-icon">
+  //             <p>...</p>
+  //           </div>
+  //         </section>
+  //       );
+  //     });
+  //   }
+  // }
+
+  //
+
+  return (
+    <div>
+      {users &&
+        users.map((user) => {
+          {
+            const name = `${user.firstName} ${user.lastName}`;
+            const initials = name
+              .match(/(\b\S)?/g)
+              .join("")
+              .match(/(^\S|\S$)?/g)
+              .join("")
+              .toUpperCase();
+          }
```

Comment:
It doesn't look like any of this is being used, and it's wrapped in curly braces for some reason? Let's remove it if it isn't being used